test/Microsoft.Sbom.Api.Tests/Converters/SbomToolManifestPathConverterTests.cs

```diff
@@ -37,85 +40,167 @@ public void Setup()
     }
 
     [TestMethod]
-    public void SbomToolManifestPathConverterTests_ValidPath_Succeeds()
+    public void SbomToolManifestPathConverterTests_ValidPath_Succeeds_Windows()
     {
+        var os = OSPlatform.Windows;
+        if (!isWindows)
```

Comment:
I think I'm nervous about the number of tests this change has caused to become conditional on OS-type. Maybe I'm thinking about the issue wrong, but if we've made things resilient shouldn't we only need to make test changes in the configSanitizerTests, one test for windows receiving linux and one for linux receiving windows?

Reply:
Paths on windows starting with "C:\" and "/home/" on linux was causing a lot of tests to fail exclusively on linux which was why I ended up splitting the tests.

Reply:
Hmm, maybe instead we could condition the `rootPath` value on the OS, that way we get to use the same test across OSs instead of maintaining X versions of it.